File: python/src/commands/hello.py
import click


@click.command()
@click.argument("name")
@click.option("--uppercase", "-u", is_flag=True, help="Print greeting in uppercase.")
def hello(name, uppercase):
    """Greet a user."""
    greeting = f"Hello, {name}!"
    click.secho(greeting.upper() if uppercase else greeting, fg="green")

File: src/commands/hello.js
import { Command } from 'commander';
import chalk from 'chalk';

export const helloCommand = new Command('hello')
  .description('Greet a user')
  .argument('<name>', 'name to greet')
  .option('-u, --uppercase', 'print greeting in uppercase')
  .action((name, options) => {
    const greeting = `Hello, ${name}!`;
    console.log(chalk.green(options.uppercase ? greeting.toUpperCase() : greeting));
  });
